tests/protocol/test_extensions_goal_state_from_extensions_config.py

```diff
@@ -45,3 +53,8 @@ def test_it_should_parse_empty_depends_on_as_dependency_level_0(self):
 
             self.assertEqual(0, extensions[0].settings[0].dependencyLevel, "Incorrect dependencyLevel}")
```

Comment:
another typo, if you want to fix it in this PR.

Reply:
I see it. fixed